cmd/internal/collector/option.go

```diff
@@ -52,6 +56,38 @@ func ParseOption() {
 	validateConfiguration()
 }
 
+// MustGetAPIKey returns the API Key from environment variables
+// if an API Key can not be find, it exits the process
+func MustGetAPIKey() string {
+	key, present := os.LookupEnv("CCLOUD_API_KEY")
+	if present && key != "" {
+		return key
+	}
+
+	key, present = os.LookupEnv("CCLOUD_USER")
+	if present && key != "" {
+		return key
+	}
```

Comment:
May want to log a warning message in this case?

Reply:
Yep, that's a good idea. I am planning to refactor the logging system at some point. I will include this warning in the next PR 👍